src/screens/Record/index.js

```diff
@@ -1,23 +1,28 @@
-import React from 'react';
+import React, {useEffect} from 'react';
 import {connect, useDispatch} from 'react-redux';
 import {View, TouchableOpacity, Text} from 'react-native';
 
-import actionsCreator from '../../redux/actions';
-
 import CardExpression from '../../components/CardExpression';
+import historyAction from '../../redux/historyApi/actions';
 
 import {styles} from './style';
 
 const mapStateToProps = state => {
-  const {history} = state;
-  return {history};
+  const {history, user, historyApi} = state;
+  return {history, user, historyApi};
 };
 
-const Record = ({history, navigation}) => {
+const Record = ({historyApi, navigation}) => {
   const dispatch = useDispatch();
 
+  useEffect(() => {
+    dispatch(historyAction.getExpressions());
+  }, [historyApi.idMsg]);
```

Comment:
This is wrong here, you should get the expressions in the edit/update action, after the success action. Doing this is not a really good approach.